Orange/widgets/visualize/owdistributions.py

```diff
@@ -148,54 +152,60 @@ def __init__(self):
         box2 = gui.indentedBox(box, sep=4)
         self.cb_rel_freq = gui.checkBox(
             box2, self, "relative_freq", "Show relative frequencies",
-            callback=self._on_relative_freq_changed)
+            callback=self._on_relative_freq_changed,
+            tooltip="Normalize probabilities so that probabilities for each group-by value sum to 1.")
         gui.separator(box2)
         self.cb_prob = gui.comboBox(
             box2, self, "show_prob", label="Show probabilities",
             orientation="horizontal",
-            callback=self._on_relative_freq_changed)
+            callback=self._on_relative_freq_changed,
+            tooltip="Show probabilities for a chosen group-by value (at each point probabilities for all group-by values sum to 1).")
 
-        plotview = pg.PlotWidget(background=None)
-        self.mainArea.layout().addWidget(plotview)
+        self.plotview = pg.PlotWidget(background=None)
+        self.mainArea.layout().addWidget(self.plotview)
         w = QtGui.QLabel()
         w.setSizePolicy(QtGui.QSizePolicy.Expanding, QtGui.QSizePolicy.Fixed)
         self.mainArea.layout().addWidget(w, Qt.AlignCenter)
-
-        self.plot = pg.PlotItem()
-        self.plot.hideButtons()
-        plotview.setCentralItem(self.plot)
+        self.ploti = pg.PlotItem()
+        self.plot = self.ploti.vb
+        self.ploti.hideButtons()
+        self.plotview.setCentralItem(self.ploti)
 
         self.plot_prob = pg.ViewBox()
-        self.plot.hideAxis('right')
-        self.plot.scene().addItem(self.plot_prob)
-        self.plot.getAxis("right").linkToView(self.plot_prob)
-        self.plot.getAxis("right").setLabel("Probability")
+        self.ploti.hideAxis('right')
+        self.ploti.scene().addItem(self.plot_prob)
+        self.ploti.getAxis("right").linkToView(self.plot_prob)
+        self.ploti.getAxis("right").setLabel("Probability")
         self.plot_prob.setZValue(10)
-        self.plot_prob.setXLink(self.plot)
+        self.plot_prob.setXLink(self.ploti)
         self.update_views()
-        self.plot.vb.sigResized.connect(self.update_views)
+        self.ploti.vb.sigResized.connect(self.update_views)
         self.plot_prob.setRange(yRange=[0,1])
 
         def disable_mouse(plot):
             plot.setMouseEnabled(False, False)
             plot.setMenuEnabled(False)
 
-        disable_mouse(self.plot.getViewBox())
+        disable_mouse(self.plot)
         disable_mouse(self.plot_prob)
 
+        self.tooltip_items = []
+        self._tooltip_delegate = HelpEventDelegate(self.help_event)
+        self.plot.scene().installEventFilter(self._tooltip_delegate)
```

Comment:
Why is `_tooltip_delegate` an instance attribute if not used anywhere else? Seems like these two could as well be a single (logical) line?